lib/worker/OperationQueue.js

```diff
@@ -0,0 +1,363 @@
+/*!
+ * Copyright (c) 2017-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const _continuityConstants = require('../continuityConstants');
+const _util = require('../util');
+const database = require('bedrock-mongodb');
+const logger = require('../logger');
+const pLimit = require('p-limit');
+
+// Controls the max concurrency for a given number of promises executing. This
+// is done to ensure we reduce pressure on the NodeJS Promise Queue and limit
+// the number concurrent I/O operations. The number 25 was arbitrarily picked.
```

Comment:
Let's do better than arbitrary. Is `25` truly out of a hat? This is used when validating operations (which we should mention). I see that we may have to validate `250` concurrently at any time, which is the default max cached queue size. Perhaps we could talk about the number `25` being a 1/10th the max number of concurrent ops being validated?   If we're going to be choosing semi-arbitrary things, I'd like us to be using powers of ten in some way because they are easier to reason about and we can at least point to that.